File: CPP/Program/BasicMove.cpp
#ifndef BASICMOVE_CPP
#define BASICMOVE_CPP

#include <iostream>
#include <string>

using namespace std;

class BasicMove{
    private:
        string name;
        int duration;
        string difficulty;
    public:
        BasicMove(string name, int duration, string difficulty){
            this->name = name;
            this->duration = duration;
            this->difficulty = difficulty;
        }

        // Setter and Getter
        void setName(string name){
            this->name = name;
        }
        string getName(){
            return this->name;
        }

        void setDuration(int duration){
            this->duration = duration;
        }
        int getDuration(){
            return this->duration;
        }

        string getDifficulty(){
            return this->difficulty;
        }
        void setDifficulty(string difficulty){
            this->difficulty = difficulty;
        }

        ~BasicMove(){}
};

#endif

File: Python/Program/BasicMove.py
class BasicMove:
    def __init__(self, name, duration, difiiculty):
        self.__name = name
        self.__duration = duration
        self.__difficulty = difiiculty
    
    def setName(self, name):
        self.__name = name
    def getName(self):
        return self.__name
    
    def setDuration(self, duration):
        self.__duration = duration
    def getDuration(self):
        return self.__duration
    
    def setDifficulty(self, difficulty):
        self.__difficulty = difficulty
    def getDifficulty(self):
        return self.__difficulty
